.gitignore

```diff
@@ -0,0 +1,2 @@
+# .gitignore
+node_modules/
```

Comment:
- [ ] I can see that you added node_modules directory but I can see that node_modules folder is already there in your repo please remove it.

Reply:
- [ ] kindly remove the node_modules said by the previous reviewer